src/exojax/spec/modit.py

```diff
@@ -467,18 +400,29 @@ def setdgm_vald_each(ielem, iion, atomicmass, ionE, dev_nu_lines, logsij0, elowe
     set_dgm_minmax = []
     Tarr_list = fT(*kargs)
     for Tarr in Tarr_list:
-        qt_284_T = vmap(interp_QT284, (0, None, None))(Tarr, T_gQT, gQT_284species)
+        qt_284_T = vmap(interp_QT284, (0, None, None))(Tarr, T_gQT,
+                                                       gQT_284species)
         SijM, ngammaLM, nsigmaDl = vald_each(Tarr, PH, PHe, PHH, R, qt_284_T, \
              QTmask, ielem, iion, atomicmass, ionE, \
                    dev_nu_lines, logsij0, elower, eupper, gamRad, gamSta, vdWdamp)
-        floop = lambda c, arr:  (c, jnp.nan_to_num(arr, nan=jnp.nanmin(arr)))
+        floop = lambda c, arr: (c, jnp.nan_to_num(arr, nan=jnp.nanmin(arr)))
         ngammaLM = scan(floop, 0, ngammaLM)[1]
-        set_dgm_minmax.append(minmax_dgmatrix(ngammaLM, res))
-    dgm_ngammaL = precompute_dgmatrix(set_dgm_minmax, res=res)
+        set_dgm_minmax.append(
+            minmax_ditgrid_matrix(ngammaLM, dit_grid_resolution))
+    dgm_ngammaL = precompute_ditgrid_matrix(
```

Comment:
precompute_ditgrid_matrix -> precompute_modit_ditgrid_matrix
[Line 276](https://github.com/HajimeKawahara/exojax/blob/16b08dc01067e52402a6a17007a955e277dff601/src/exojax/spec/modit.py#L276) (in set_ditgrid_matrix_hitran) may be the same.